src/components/grid/tile/tile.reducer.ts

```diff
@@ -1,27 +1,27 @@
-import { Actions, ActionTypes } from './tile.actions'
```

Comment:
delete file?

Reply:
please delete 😄 I didn't because you were working on it, but I don't think it's necessary now